data/SmithMartin_2020/metadata.yml

```diff
@@ -0,0 +1,164 @@
+    the xylem. ^p Due to the very small size of leaves in some species (including
+    their reduction to scales in Callitris and Allocasuarina), it was not possible
+    to employ the optical method in all these species. Hence, leaf vulnerability to
+    water deficit curves was extracted from Skelton, Brodribb, McAdam, et al. (2017),
+    where the same plant community of trees and shrubs was measured in 2016. In this
+    study, leaf hydraulic conductance (kleaf) was measured using the rehydration kinetics
+    method on whole leaves or in the case of microphyllous leaves or needle leaves
+    small terminal shoots were measured (Skelton, Brodribb, McAdam, et al., 2017).
+    We extracted the xylem water potentials associated with 12% (P12 kleaf, MPa) and
+    50% (P50 kleaf, MPa) loss of kleaf. ^p HSM were calculated as the difference between
+    midday leaf water potential (minimum water potential) and P12 (P12 HSM), P50 (P50
+    HSM) and P88 (P88 HSM) embolism extracted from OVC. Midday leaf water potential
+    was taken from Skelton, Brodribb, McAdam, et al. (2017) when water potential was
+    measured on the same plant community during the end of a severe drought caused
+    by the 2015–2016 El Niño Pacific southern oscillation (Table S1).
+  original_file: Archived in Google Drive
+  notes: .na
+locations:
+  Tasmanian dry sclerophyll woodland:
+    latitude (deg): 42.048
+    longitude (deg): 147.957
+    elevation (m): 134
+    soil type: Podzolic soils formed from Jurassic Dolerite parent material (Skelton,
+      Brodribb, McAdam, et al., 2017)
+    precipitation, MAP (mm): 600--700
+    precipitation, description: Rainfall is sporadic throughout the year without a
+      clear wet or dry season and mean annual rainfall at a nearby location (Swansea,
+      <10 km away) is ~600–700 mm (Australian Bureau of Meteorology).
+    vegetation type: Evergreen dry sclerophyll woodland
+    description: This study was conducted on the east coast of Tasmania, Australia.
+      Vegetation at our study site is evergreen dry sclerophyll woodland. Woodlands
+      originally covered close to 30% of the Australian continent including Tasmania
+      and are often dominated by Callitris spp., Casuarina spp., Eucalyptus spp. and
+      Melaleuca spp. (Lindenmayer, Crane, & Michael, 2005). At the study site, the
+      woody vegetation consisted of two canopy layers - a 5–20 m overstory and a 1–3
+      m understory (Skelton, Brodribb, McAdam, et al., 2017). We selected the eight
+      dominant woody species from this site, which consisted of four canopy trees
+      species that included one gymnosperm and three angiosperm and four angiosperm
+      understory shrubs (Table 1).
+contexts: .na
+traits:
+- var_in: Type
+  unit_in: .na
+  trait_name: plant_growth_form
+  entity_type: species
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: .na
+  methods: unknown
+- var_in: HSM_50
```

Comment:
The data columns P50_MPa & P88_Mpa are also traits in AusTraits (water_potential_50percent_lost_conductivity & water_potential_88percent_lost_conductivity); I've gone ahead and added those in. And also created a new trait for hydraulic safety margin at P88, as other studies had that as well.